var/spack/repos/builtin/packages/madgraph5amc/package.py

```diff
@@ -57,6 +57,42 @@ class Madgraph5amc(MakefilePackage):
     variant("ninja", default=False, description="Use external installation" + " of Ninja")
     variant("collier", default=False, description="Use external installation" + " of Collier")
     variant("pythia8", default=False, description="Use external installation of Pythia8")
+    variant(
+        "models",
+        values=(
+            "2HDM",
+            "4Gen",
+            "DY_SM",
+            "EWdim6",
+            "IDM_NLO_EW_CM_UFO",
+            "RS",
+            "SMScalars",
+            "TopEffTh",
+            "heft",
+            "heft_v4",
+            "loop_qcd_qed_sm",
+            "loop_qcd_qed_sm_4FS",
+            "loop_qcd_qed_sm_Gmu",
+            "loop_qcd_qed_sm_Gmu_4FS",
+            "loop_qcd_qed_sm_Gmu_4FS_ew",
+            "loop_qcd_qed_sm_a0",
+            "loop_qcd_qed_sm_forSUSY",
+            "mssm_v4",
+            "nmssm",
+            "sextet_diquarks",
+            "sm_v4",
+            "top-philic_NLO_EW_CM_UFO",
+            "triplet_diquarks",
+            "usrmod_v4",
+            "uutt_sch_4fermion",
+            "uutt_tch_scalar",
+        ),
+        default=[],
```

Comment:
(Untested) To pass audit checks, this should probably be:
```suggestion
        default=None,
```

Reply:
Let's see if this works.

Reply:
Seems my suggestion was bad :-D sorry